src/coreComponents/linearAlgebra/interfaces/hypre/HypreVector.hpp

```diff
@@ -244,29 +191,25 @@ class HypreVector final : private VectorBase< HypreVector >
 
   ///@}
 
+private:
+
+  /// ParVector pointer alias
+  using HYPRE_ParVector = hypre_ParVector *;
+
+public:
+
   /**
    * @brief Returns a pointer to the implementation.
    * @return the underlying HYPRE_ParVector object.
    */
   HYPRE_ParVector const & unwrapped() const;
 
-  /**
-   * @brief Returns a pointer to the implementation.
-   * @return the underlying HYPRE_IJVector object.
-   */
-  HYPRE_IJVector const & unwrappedIJ() const;
-
 private:
 
-  /**
-   * Pointer to underlying HYPRE_IJVector type.
-   */
-  HYPRE_IJVector m_ij_vector;
-
   /**
    * Pointer to underlying HYPRE_ParVector type.
    */
-  HYPRE_ParVector m_par_vector;
+  HYPRE_ParVector m_vec;
```

Comment:
Is there a specific reason to keep using `HYPRE_ParVector`? Given the ongoing refactoring, would it make sense to use directly `hypre_ParVector`? Also, can we now declare `m_vec` as unique pointer?  

```c++
std::unique_ptr< hypre_ParVector > m_vec;
```